docs/paper/paper.md

```diff
@@ -32,12 +32,14 @@ In contrast to existing methods, the `flory` package implements a state-of-art m
 Finding coexisting phases is a general task in many fields, such as chemical engineering [@lukas2007Computational] and soft matter physics [@jacobs2023Theory].
 There are several strategies to theoretically predict coexisting phases [@jacobs2023Theory], including direct spatially-resolved simulations [@shrinivas2021Phase], the construction of the convex hull of the free energy  [@mao2019Phase], solving the balance equations [@zwicker2022Evolved], and direct minimization of the free energy [@lukas2007Computational].
 However, there are only a few open-source packages that implement these strategies.
-Most notably, `pycalphad` [@otis2017Pycalphad] and `OpenCalphad` [@sundman2015Implementation] combine a database to compute phase diagrams (Calphad) of mixtures with few components. <!-- What do they combine? I expect two aspects that are combined -->
+Most notably, `pycalphad` [@otis2017Pycalphad] and `OpenCalphad` [@sundman2015Implementation] combine a database of phases several strategies above to compute phase diagrams (Calphad) of mixtures with few components.
```

Comment:
I don't get this sentence. What does Calphad actually do? Is it a minimization technique or a database of known phases, or a combination? We might want to discuss this in person :)

Reply:
Sorry I missed a word there. It's a combination of candidate phases and minimization technique.